content/documentation/zooming-images.md

```diff
@@ -0,0 +1,207 @@
+
+## Display IIIF Images in Your Project
+
+To display a IIIF image in your project, you need to point to a IIIF `info.json` file for the image. Either one you’ve identified from an [external source](#use-existing-iiif-images), or one you’ve [created yourself](#creating-your-own-iiif-images) with `quire process --iiif`. If it is an image you processed yourself and it is still in the `processed/` directory, the path to the JSON file would be `/img/iiif/processed/FILENAME/info.json`.
+
+Along with the path to the `info.json` file, you also need to include `media_type: "iiif"` and a path to a lower-res version of the image, hosted in your project’s `static/img` directory (which again is used for Quire’s PDF and e-book outputs). All of this goes in your `figures.yml` file.
+
+```yaml
+- id: "irises"
+  src: "figures/irises.jpg"
+  media_type: "iiif"
+  iiif: "https://data.getty.edu/museum/api/iiif/671108/info.json"
+```
+
+The image can then be added to your Markdown files using the `q-figure-zoom` shortcode. This will display the static image on the page, and then when clicked, will open the figure viewer for the fully zoomable IIIF version.
+
+```go
+{{</* q-figure-zoom id="irises" */>}}
+```
+
+IIIF Images can also be displayed in the built-in image viewer on [entry pages](/documentation/pages). Do this by including the appropriate `id` in the [object data](/documentation/collection-catalogues/#capture-object-data) in your `objects.yml` file.
+
+```yaml
+object_list:
+  - id: 1
+    title: *Irises*
+    artist: Vincent van Gogh
+    year: 1889
+    medium: Oil on canvas
+    dimensions: 74.3 × 94.3 cm (29 1/4 × 37 1/8 in.)
+    location: J. Paul Getty Museum
+    link: https://www.getty.edu/art/collection/objects/826/vincent-van-gogh-irises-dutch-1889/
+    figure:
+      - id: "irises"
+```
+
+### Include a Static Fallback Image
+
+The static fallback image is required for displaying a version of the IIIF image outside of the zooming image viewer on the online version of your project, as well as for Quire’s PDF and e-book outputs. This is the standard `src` attribute in your `figures.yml` listing.
+
+```yaml
+src: "figures/irises.jpg"
+```
+
+Like with other non-zooming figure images, these static images for IIIF would typically be a JPEG of around 1,800px on the longest side, and at 70–80% quality. These specs provide both a decent web experience and work for printing in the PDF, without being too large of a file size. Read more in the [*Figure Images*](/documentation/figure-images/) chapter of the documentation.
```

Comment:
This repeats the "prepare images" section in reference to smaller image sizes.  Rather than repeat, maybe in one of these two sections we just say "see Figure Images section for more," etc. 

And, I'll make a note to update that page from 1200 pixels to 1800.

Reply:
Yes, it's another case of redundancy. Primarily because there's a good chance people might not read both sections (depending on whether or not they are using existing IIIF images). And to me it's such a little piece of information, I didn't want people to have to click through and go looking for it.